README.md

```diff
@@ -16,288 +16,116 @@
-
-### Data Wrangling
-Courses | Duration | Effort
-:-- | :--: | :--:
-[Data Wrangling with MongoDB](https://www.udacity.com/course/data-wrangling-with-mongodb--ud032)| 8 weeks | 10 hours/week
+[What is Data Science](https://www.coursera.org/learn/what-is-datascience)
 
-### Big Data
-Courses | Duration | Effort
-:-- | :--: | :--:
-[Intro to Hadoop and MapReduce](https://www.udacity.com/course/intro-to-hadoop-and-mapreduce--ud617)| 4 weeks | 6 hours/week
-[Deploying a Hadoop Cluster](https://www.udacity.com/course/deploying-a-hadoop-cluster--ud1000)| 3 weeks | 6 hours/week
-
-### Database
-Courses | Duration | Effort
-:-- | :--: | :--:
-[Stanford's Database course](https://lagunita.stanford.edu/courses/DB/2014/SelfPaced/about)| - weeks | 8-12 hours/week
-
-### Natural Language Processing
-Courses | Duration | Effort
-:-- | :--: | :--:
-[Deep Learning for Natural Language Processing](http://cs224d.stanford.edu/)| - weeks | - hours/week
-
-### Deep Learning
-Courses | Duration | Effort
-:-- | :--: | :--:
-[Deep Learning](https://www.udacity.com/course/deep-learning--ud730)| 12 weeks | 8-12 hours/week
-
-### Capstone Project
-- Participate in Kaggle competition
-- List down other ideas
-
-
-### Specializations
+### Single Variable Calculus
+[Calculus 1A: Differentiation](https://www.edx.org/course/calculus-1a-differentiation-mitx-18-01-1x)
+[Calculus 1B: Integration](https://www.edx.org/course/calculus-1b-integration-mitx-18-01-2x)
+[Calculus 1C: Coordinate Systems & Infinite Series](https://www.edx.org/course/calculus-1c-coordinate-systems-infinite-mitx-18-01-3x)
 
-After finishing the courses above, start your specializations on the topics that you have more interest.
-You can view a list of available specializations [here](https://github.com/open-source-society/data-science/blob/master/extras/specializations.md).
+### Linear Algebra
+[Essence of Linear Algebra](https://www.youtube.com/playlist?list=PLZHQObOWTQDPD3MizzM2xVFitgF8hE_ab)
+[Linear Algebra](https://ocw.mit.edu/courses/mathematics/18-06sc-linear-algebra-fall-2011/)
 
-![keep learning](http://i.imgur.com/REQK0VU.jpg)
+### Multivariable Calculus
+[Multivariable Calculus](http://ocw.mit.edu/courses/mathematics/18-02sc-multivariable-calculus-fall-2010/index.htm)
+
+### Introduction to Computer Science
+Students who already know basic programming in any language can skip this course
+[Python for Everybody](https://www.py4e.com/) 
+[Introduction to Computer Science and Programming Using Python](https://www.edx.org/course/introduction-computer-science-mitx-6-00-1x-7)
+[Introduction to Computational Thinking and Data Science](https://www.edx.org/course/introduction-computational-thinking-data-mitx-6-00-2x-3)
+
+### Data Structures and Algorithms
+[Algorithms, Part I](https://www.coursera.org/learn/algorithms-part1)
+[Algorithms, Part II](https://www.coursera.org/learn/algorithms-part2)
```

Comment:
These courses are completely in Java and steep learning curve from the start. Those coming straight from python/r/julia will have a hard time adjusting to both the course materials and programming syntax. Suggest an optional course on Java as pre-requisite, specifically (Java Programming Part 1 and 2)[https://java-programming.mooc.fi/] by University of Helsinki. Gives college credit for Finland residents.

Reply:
@mattjperez Cogent comment. Can you comment [in the issue](https://github.com/ossu/data-science/issues/61)?